bci_essentials/bci_data.py

```diff
@@ -647,8 +787,44 @@ def mne_export_resting_state_as_raw(self):
     # SIGNAL PROCESSING
     # Preprocessing goes here (windows are nchannels by nsamples)
     def preprocessing(self, window, option=None, order=5, fc=60, fl=10, fh=50):
-        """
-        Howdy
+        """Signal preprocessing.
```

Comment:
These values are missing the optional label to them

Reply:
Fixed